### ISSUE TITLE
Feeding trough not breeding animals

### ISSUE THREAD

**Issue by Dezzie9889** [blay09/FarmingForBlockheads#192]
### Minecraft Version

1.20.x

### Mod Loader

Forge

### Mod Loader Version

47.1.3

### Mod Version

14.0.1

### Balm Version

7.0.4

### Describe the Issue

Animals will produce hearts when using feeding trough but do not produce a child when feeding from trough.

**Comment by HumdrumPanic** [comment]
How many animals of a type are you trying to breed at once? it seems it may need more than two of a type of animal in order to breed.

Been tracking something that may be similar/related :
https://legacy.curseforge.com/minecraft/mc-mods/resource-chickens/issues/18

**Comment by Nyxsohn** [comment]
We are seeing a similar issue, just noticed today.
MC: 1.20.1
Forge: 47.1.0
Farming for Blockheads: 14.0.0
Biome: Mushroom Fields
Config: 8 block and max 24 animal
What we're seeing: Example cow/mooshroom cow. One trough, two cows in range. One cow will feed, explosion effect, hearts. Second cow will never feed. 
We tried various directions (cows north or east of trough), we were using sheared mooshrooms so went OP and spawned in new cows, etc. Same no matter what.
Then, we tried with our pigs, same issue. One feeds, the other ignores entirely. 
So next we tried with 4 animals. As Humdrum mentioned, this does work. So we need 3+ for the trough to work.
